### ISSUE TITLE
"Contains" rules filter for fields with more than one value?

### ISSUE THREAD

**Issue by khromov** [pocketbase/pocketbase#229]
👋 Let's say you have a "chat" collection with more than one "participants" (MAX SELECT = 100) and you'd like people to be able to see the chat collection **only if they are participating in it.**

Is it safe to make a rule query such as `participants ~ @request.user.id`? Will PocketBase internally use a `LIKE %userId%` query, and is there a risk that it will evaluate incorrectly? (For example if one user has the id `1` and another has the id `11`?

**Comment by ganigeorgiev** [comment]
Yes, when `~` or `!~` operators are used we are converting then to `LIKE` and `NOT LIKE` expressions.
The user id is not incremental, it is a 15 chars unique random string and there are no plans for now to change that, so in general it should be safe to use the "~" operator in your case.

For your example, I would only add a check that there is actually an authenticated user, eg.:
```
@request.user.id != '' && participants ~ @request.user.id
```

In future versions I may consider making the filter a little more smarter and in case a relation field is checked it'll be converted to a `json_each` or other similar SQLite function.

_I'll move the issue to Q&A discussion since the format is more suitable for that._
